ldms/src/core/ldms.c

```diff
@@ -1648,7 +1654,6 @@ void __make_mdesc(ldms_mdesc_t vd, ldms_mdef_t md, off_t *value_off)
 		(*value_off) += __ldms_value_size_get(md->type, md->count);
 	} else
 		vd->vd_data_offset = 0; /* set after all metrics defined */
-
 }
```

Comment:
Right after the function __make_mdesc add this pragma to restore normal diagnostics:

#pragma GCC diagnostic pop